### PR TITLE
PRO-4734 update java

### PR DESCRIPTION
**Pull request by damongreen123**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM hmcts/cnp-java-base:openjdk-8u181-jre-alpine3.8-1.0
+FROM hmcts/cnp-java-base:openjdk-8u191-jre-alpine3.9-1.0
 
 # Mandatory!
 ENV APP probate-orchestrator-service.jar
```
Comment (KevinJCross) [inline Dockerfile]: this is not needed any more :smile:

File: Dockerfile [hunk 2]
```diff
@@ -12,4 +12,4 @@ HEALTHCHECK --interval=10s --timeout=10s --retries=10 CMD http_proxy="" wget -q
 
 EXPOSE 8888
 
-ENTRYPOINT [ "/entrypoint.sh" ]
\ No newline at end of file
+ENTRYPOINT [ "/entrypoint.sh" ]
```
Comment (KevinJCross) [inline Dockerfile]: this has to change to `CMD [ "probate-orchestrator-service.jar" ]`

File: Dockerfile (hunk 1 above)
Comment (KevinJCross) [reply Dockerfile]: see https://github.com/hmcts/cmc-claim-store/pull/733

Comment (damongreen123) [reply Dockerfile]: Thanks Kevin, made some changes.

**Pull request merged**